docs/releasing.md

```diff
@@ -4,7 +4,26 @@
 
 ### Cadence
 
-We ship once a month, on the Thursday before the 1st. While not necessary, followup minor/patch releases may be done if warranted. The planned ship dates are added to the internal Lighthouse calendar.
+We aim to release every 3 weeks. Our schedule is set as follows: One day before the [expected Chromium branch point](https://www.chromium.org/developers/calendar) (which is every six weeks) and again exactly 3 weeks after that day.
+
+Example, the next fews releases (as of this writing) are:
+
+* _Oct 15 2019_
+* Nov 5 2019
+* _Nov 26 2019_
+* Dec 17 2020
+* _Jan 7 2020_
+* Jan 28 2020
+* _Feb 18 2020_
+* Mar 10 2020
+
+Italicized dates are the day before the expected Chromium branch point.
+
+The planned ship dates are added to the internal Lighthouse calendar.
```

Comment:
This seems weird to have on a "public-ish" doc.  I think it's on us to handle this.

Unless we ship a public LH release calender @egsweeny

Reply:
I could put a big disclaimer here, saying that most of this doc is for internal, core member only. Just because that's the case, doesn't mean we need to keep the information private - I think keeping this stuff centralized to the repo is better than in a google doc.